### DISCUSSION TITLE
installation failure

### DISCUSSION THREAD

**Discussion by Silencewu7** [zqfang/GSEApy#290] [Q&A]
pip install gseapy
......
`ERROR: Failed building wheel for gseapy
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (gseapy)`

I‘ve upgraded my pip and installed 'pyproject', but it still doesn't work. How should I deal with it.

**Comment by zqfang** [comment]
what operation system you are trying to install ?

**Reply by Silencewu7** [reply]
windows,and RUST compiler had been installed beforehand

**Reply by Silencewu7** [reply]
Can this package be supported by python 3.9?

**Reply by zqfang** [reply]
It suppose to support windows with py3.9. 

Can you try 

```
pip install gseapy==1.1.3
```

The windows binaries are failed to build for the lastest version. I will try to fix it

**Reply by zqfang** [reply]
hi @Silencewu7 , your issue should be gone now. I have upload windows binaries for the latest version of GSEApy. You don't need to install rust compiler any more.

**Reply by Silencewu7** [reply]
have successfully installed it! many thanks to u!
